<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1170</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1170</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>人脸识别</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/img/images/moon.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>1151</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;新宋体&quot;;
background-color: rgb(222, 222, 222);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>底库创建</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>图片路径:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>底库保存路径:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>89</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>底库处理线程数:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imageFromPathLineEdit">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>30</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dbCreateThreadCount">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dbSavePathLineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imageFromSelectPushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dbSavePathSelectPushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>选择..</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="createDBProgressBar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>1031</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="createDBDisplayInfoLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>971</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>点击下面的开始进行创建</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createDBStartButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>1:1比对</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>VS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="similarityLabel">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 12pt &quot;Arial&quot;;
color: rgb(255, 0, 127);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compareButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>190</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>比对</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="leftImageButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>221</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="rightImageButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>221</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>1:N 比对</string>
     </attribute>
     <widget class="QPushButton" name="nCompareLoadDestDBButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>加载</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>32</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>图片库:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nCompareDestDBLineEdit">
      <property name="geometry">
       <rect>
        <x>131</x>
        <y>30</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nCompareSrcImageButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>151</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="nCompareCompareButton">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>比对Top5</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="nCompareMinScoreLineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>70</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>151</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>最低阀值:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>269</y>
        <width>931</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Top5</string>
      </property>
      <widget class="QPushButton" name="nCompareTop1ImageLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nCompareTop2ImageLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>0</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nCompareTop3ImageLabel">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nCompareTop4ImageLabel">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>0</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="nCompareTop5ImageLabel">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>0</y>
         <width>151</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/img/images/uploadImg.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="nCompareTop1ScoreLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nCompareTop2ScoreLabel">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nCompareTop3ScoreLabel">
       <property name="geometry">
        <rect>
         <x>432</x>
         <y>151</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nCompareTop5ScoreLabel">
       <property name="geometry">
        <rect>
         <x>771</x>
         <y>149</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nCompareTop4ScoreLabel">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>150</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前底库人数:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nComareDestDBCountLabel">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>系统设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1111</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238);</string>
      </property>
      <property name="title">
       <string>设置</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>人脸引擎:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>SeetaFacer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VIFacer</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
